languages/python/default.nix

```diff
@@ -49,6 +49,79 @@ in {
       defaultText = lib.literalExpression "pkgs: [ ]";
       example = lib.literalExpression "pkgs: [ pkgs.requests ]";
     };
+    config = lib.mkOption {
```

Comment:
Are you sure we want to call it “config”? Granted I also have `ghciConfig` but maybe that should be renamed too, now that I think about it.

Reply:
Maybe we could name it `initExtra` (or `init` since the module doesn't write anything to the file) to match the other shell HM options (bash, zsh...)?

Reply:
By the way, it seems that in HM the convention is to name `configFile` the variable for the configuration file path. We could rename `configPath` to `configFile` too.

Reply:
Good idea.